jwst/cube_build/src/cube_dq_utils.c

```diff
@@ -765,7 +774,12 @@ int dq_nirspec(int overlap_partial,
   long istart, in, iend, ii, i;
   double c1_min, c2_min, c1_max, c2_max;
   int *idqv ;  // int vector for spaxel
+
   idqv = (int*)calloc(ncube, sizeof(int));
+  if (NULL==idqv)
```

Comment:
Nitpick: I am used to constant on RHS.
```suggestion
  if (idqv == NULL)
```

Reply:
In C constants should go on the LHS in logical operations to prevent accidental variable setting.  The conditional `if(idqv=NULL)` is valid, will compile, then at run time sets `idqv` to `NULL`, causing a memory leak.  Writing `if(NULL=idqv)` is invalid syntax and will cause a compilation error.